static/js/tools/statsvar_menu.test.js

```diff
@@ -40,10 +40,12 @@ it("filtering the menu", () => {
   act(() => {
     render(
       <Menu
-        selectedNodePaths={[[0, 0]]}
+        selectedNodePaths={[["0", "0"]]}
```

Comment:
what is the reason to change from number to string?

Reply:
Previously we have both string and number for node paths. e.g, the node path parsed from url is string and the node path built from index is number. To be consistent, I changed everything to string, because string is more general, it would be easier to change if we want to modify the representation of node path in the future.